src/js/components/FileInput/FileInput.js

```diff
@@ -120,6 +128,20 @@ const FileInput = forwardRef(
       }
     }
 
+    const parseBytes = (bytes) => {
```

Comment:
I think we should call this `normalizeBytes` and pull it outside of the component, perhaps around line 18.

Reply:
On second thought, maybe we should omit the object value to start with. We can always add it in later as needed.

Reply:
Gotcha, I'll leave it commented out where you recommended and down the line we can reintroduce it!